### PR TITLE
BUILD-353: update with latest CRD,group,csi driver renames

### PR DESCRIPTION
**Pull request by gabemontero**
Update on the names or the various CRD/group/driver names for the sharedconfigmap / sharedsecret / sharedresource following the meeting between myself and 
@deads2k 
@adambkaplan 
@bparees 
@jsafrane 
@coreydaley 

on Sep 24 2021

@sbose78 @siamaksade FYI

### CODE REVIEW

**Review by adambkaplan — APPROVED** [review]
/approve

Want someone to double-check, I think this looks right.

**Comment by gabemontero** [comment]
> /approve
> 
> Want someone to double-check, I think this looks right.

/assign @coreydaley 

please provide the double-check @adambkaplan asks .... mostly lines up with your openshift/api work

File: enhancements/cluster-scope-secret-volumes/csi-driver-host-injections.md [hunk 1]
```diff
@@ -184,44 +194,55 @@ And sets up ACL/RBAC to control which `ServiceAccounts` can use such objects to
...
 ```yaml
 rules:
 - apiGroups:
-  - storage.openshift.io
+  - sharedresource.openshift.io
   resources:
-  - shareresources
+  - sharedsecrets
+  - sharedconfigmaps
   verbs:
```
Comment (coreydaley) [inline enhancements/cluster-scope-secret-volumes/csi-driver-host-injections.md:227]: Should we add `watch` here also?

**Pull request merged**